trymito.io/components/CaseStudyCard/CaseStudies.tsx

```diff
@@ -0,0 +1,97 @@
+import React, { useEffect, useState } from 'react';
+import Image from 'next/image';
+import caseStudiesStyles from './CaseStudies.module.css'
+import CaseStudyCard from './CaseStudyCard';
+import caseStudyCardStyle from './CaseStudyCard.module.css'
+import { classNames } from '../../utils/classNames';
+
+const CaseStudies = (props: {}): JSX.Element => {
+
+    return (
+        <>
+            <table className={classNames(caseStudiesStyles.case_studies_table, 'only-on-desktop')}>
+                <tbody>
+                    <tr>
+                        <td>
+                            <CaseStudyCard 
+                                imageSrc='/case-studies/Top 10 bank automated critical monthly reports.png'
```

Comment:
The text on this one looks squeezed or like stretched.
<img width="568" alt="Screen Shot 2024-02-27 at 3 09 57 PM" src="https://github.com/mito-ds/mito/assets/21002431/0bff2baf-3826-4551-ab9e-015ef951abea">

Reply:
On both mobile and on the desktop